bom/pom.xml

```diff
@@ -68,6 +68,11 @@
         <artifactId>camunda-engine-spring</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.camunda.bpm</groupId>
+        <artifactId>camunda-engine-spring-6</artifactId>
```

Comment:
We have decided to not make Spring 6 artifact the default one. Strongest arguments was to prevent migration efforts for the users using AS (the majority) that still support only Spring 5. Link to decision: https://github.com/camunda/camunda-bpm-platform/issues/2755#issuecomment-1569576945
The users will be more motivated to make migration efforts when they want to adopt Spring 6 to their scenarios.